### PR TITLE
CDAP-12002 reduce default time to 5 seconds

### PR DESCRIPTION
**Pull request by rohitsinha54**
Relates to https://github.com/caskdata/cdap/pull/9651#issuecomment-331353748

### CODE REVIEW

**Comment by anwar6953** [comment]
to 5 seconds or 5ms?

File: cdap-common/src/main/resources/cdap-default.xml [hunk 1]
```diff
@@ -3106,7 +3106,7 @@
 
   <property>
     <name>security.authorization.extension.operation.time.warn.threshold.ms</name>
-    <value>30000</value>
+    <value>3000</value>
```
Comment (anew) [inline cdap-common/src/main/resources/cdap-default.xml]: hmmm that is not 5 sec

Comment (rohitsinha54) [reply cdap-common/src/main/resources/cdap-default.xml]: :)

**Comment by anew** [comment]
oh I must have looked at an outdated version. LGTM

**Comment by sreevatsanraman** [comment]
Update the md5 for docs. otherwise the docs build will fail

**Comment by rohitsinha54** [comment]
@anew : Nope. I did pushed that 😭

**Comment by sreevatsanraman** [comment]
LGTM

**Comment by rohitsinha54** [comment]
local build passed merging.

**Pull request merged**